pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -229,7 +229,11 @@ func buildHTTPMixerFilterConfig(mesh *meshconfig.MeshConfig, role model.Proxy, n
 
 	mxConfig := &mccpb.HttpClientConfig{
 		MixerAttributes: &mpb.Attributes{
-			Attributes: map[string]*mpb.Attributes_AttributeValue{},
+			Attributes: map[string]*mpb.Attributes_AttributeValue{
+				"context.reporter.uid": {
```

Comment:
nit: define a const for this string value (and local).

Reply:
it's easier to parse when the literal attribute names are inlined. these should be configurable anyways, so a const string is not perfect either.